app/Http/Requests/ProgramEditRequest.php

```diff
@@ -60,6 +61,7 @@ protected function prepareForValidation()
             'delivery_distance' => isset($this->delivery_distance) ? $this->toBoolean($this->delivery_distance) : null,
             'delivery_in_class' => isset($this->delivery_in_class) ? $this->toBoolean($this->delivery_in_class) : null,
             'work_experience_required' => isset($this->work_experience_required) ? $this->toBoolean($this->work_experience_required) : null,
+            'program_graduate' => isset($this->program_graduate) ? $this->toBoolean($this->program_graduate) : null,
```

Comment:
'program_graduate' => isset($this->program_graduate) ? $this->toBoolean($this->program_graduate) : false,